examples/armv4t_multicore/gdb.rs

```diff
@@ -272,3 +280,20 @@ impl target::ext::breakpoints::HwWatchpoint for Emu {
         Ok(true)
     }
 }
+
+impl ThreadExtraInfo for Emu {
+    fn extra_info(&self, tid: Tid, buf: &mut [u8]) -> TargetResult<usize, Self> {
+        let cpu_id = tid_to_cpuid(tid).map_err(TargetError::Fatal)?;
+        let info = format!("CPU {:?}", cpu_id);
+
+        Ok(copy_to_buf(info.as_bytes(), buf))
+    }
+}
+
+/// Copy all bytes of `data` to `buf`.
+/// Return the size of data copied.
+pub fn copy_to_buf(data: &[u8], buf: &mut [u8]) -> usize {
```

Comment:
I believe this function should already be floating around somewhere in armv4t... no need to redefine it.

Reply:
ahh, indeed: examples/armv4t/gdb/mod.rs:39

Reply:
There was a copy in armv4t, but not armv4t_multicore

Reply:
ahh, didn't realize this was `armv4t_multicore`, oops 😅 
All good